.travis.yml

```diff
@@ -0,0 +1,60 @@
+# language: node_js
```

Comment:
Is it normal that all this file is commented?

Reply:
Yep, I enabled travis when I created the repo, but for it to work I need to configure it a bit it seems. So I just commented the whole travis config file for now.